app/views/articles/new.html.haml

```diff
@@ -0,0 +1,11 @@
+= form_with model: Article.new, url: articles_path, local: true do |form|
+  %p
+    = form.label :title
+    %br/
+    = form.text_field :title, id: :article_title
+  %p
+    = form.label :content
+    %br/
+    = form.text_area :content, id: :article_content
+  %p
+    = form.submit placeholder: "Create Article"
```

Comment:
Not sure if you should use `placeholder:` here.  `form_with` is relatively new to me, but I usually use `value:`

Reply:
They both work but I will change to value for fear of the unkown